src/test/java/com/google/api/codegen/testdata/php_test_library.baseline

```diff
@@ -152,6 +155,43 @@ class LibraryServiceClientTest extends PHPUnit_Framework_TestCase
     /**
      * @test
      */
+    public function createShelfExceptionTest()
+    {
+        $grpcStub = $this->createStub([$this, 'createMockLibraryServiceImpl']);
+        $client = $this->createClient('createLibraryServiceStubFunction', $grpcStub);
+
+        $this->assertTrue($grpcStub->isExhausted());
+
+        $status = new stdClass();
+        $status->code = Grpc\STATUS_INTERNAL;
+        $status->details = 'internal error';
+        $grpcStub->addResponse(null, $status);
+
+        // Mock request
+        $shelf = new Shelf();
+
+        try {
+            $client->createShelf($shelf);
+            // If the $client method call did not throw, fail the test
+            $this->fail("Expected an ApiException, but no exception was thrown.");
+        } catch (ApiException $ex) {
+            $this->assertEquals($status->code, $ex->getCode());
+            $this->assertEquals($status->details, $ex->getMessage());
+            $actualRequests = $grpcStub->getReceivedCalls();
+            $this->assertSame(1, count($actualRequests));
+            $actualFuncCall = $actualRequests[0]->getFuncCall();
```

Comment:
I feel this function call check is redundant, it has been checked in the previous test and the method call is the same for the exception test. Therefore we should just focus on the exception handling part.